### ISSUE TITLE
Sweep: Gaussian+Global+CE, sweeping RF, training for 1M grad steps

### ISSUE THREAD

**Issue by RylanSchaeffer** [RylanSchaeffer/FieteLab-No-Free-Lunch#4]
Experiment: Train models on Gaussian place cells with global normalization and cross entropy loss, sweeping receptive field from 0.01m to 0.05m, all using ReLU. These models were trained for much longer than normal: 20 "epochs" of 50k gradient steps with batch size 200 for a total of 1M gradient steps and 200M total sampled trajectories.

W&B Page: https://wandb.ai/rylan/mec-hpc-investigations/sweeps/ea06fmvq?workspace=user-rylan

Warning: Many of these runs are still coming in. This issue contains preliminary results and will be subsequently updated.

X axis is epoch number.

Log loss:

![image](https://user-images.githubusercontent.com/8942987/161577585-eb537071-156e-452c-ad65-753be4e02315.png)

60d grid scores 

![image](https://user-images.githubusercontent.com/8942987/161577693-50552528-0b7c-4f6a-811e-76fb9ed6bf33.png)

90d grid scores

![image](https://user-images.githubusercontent.com/8942987/161577616-9a19d6ca-3863-4b63-8605-f590384ac792.png)
